### PR TITLE
Sscs 11348 ut decision refused

### PR DESCRIPTION
**Pull request by paul-pearson**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SSCS-11348

### Change description ###

ut decision refused

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/PostHearing.java [hunk 1]
```diff
@@ -83,6 +86,21 @@ public LibertyToApply getLibertyToApply() {
         return libertyToApply;
     }
 
+//    @JsonIgnore
+//    public UpperTribunalDecision getUpperTribunalDecision() {
+//        if (isNull(upperTribunalDecision)) {
+//            upperTribunalDecision = new UpperTribunalDecision();
+//        }
```
Comment (nickhill111) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/PostHearing.java:93]: remove commented code

**Review by nickhill111 — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/PostHearing.java (hunk 1 above)
Comment (paul-pearson) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/PostHearing.java]: Thanks, missed that one.

**Review by nickhill111 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/callback/DocumentType.java [hunk 2]
```diff
@@ -51,6 +51,8 @@ public enum DocumentType {
     STATEMENT_OF_REASONS_REFUSED("statementOfReasonsRefused", "SOR refused decision notice"),
     STATEMENT_OF_REASONS_APPLICATION("statementOfReasonsApplication", "Statement of Reasons Application"),
     TL1_FORM("tl1Form", "TL1 Form"),
+    UPPER_TRIBUNALS_DECISION_REMADE("upperTribunalsDecisionRemade", "Upper Tribunals decision remade"),
+    UPPER_TRIBUNALS_DECISION_REFUSED("upperTribunalsDecisionRefused", "Upper Tribunals decision refused"),
```
Comment (nickhill111) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/callback/DocumentType.java:55]: Capitalise the labels

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SendToFirstTierActions.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+    DECISION_REMADE("remade", "Decision remade by UT", EventType.UPPER_TRIBUNAL_DECISION, "Decision remade by UT", "Decision remade by UT", null),
+    DECISION_REFUSED("refused", "Decision refused by UT", EventType.UPPER_TRIBUNAL_DECISION, "Decision refused by UT", "Decision refused by UT", null);
+
+    private final String ccdDefinition;
+    private final String descriptionEn;
+    private final EventType callbackEvent;
+    private final String callbackSummary;
+    private final String callbackDescription;
+    private final DwpState postCallbackDwpState;
```
Comment (nickhill111) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SendToFirstTierActions.java:19]: set this to null here and remove null when creating the enum values

**Review by nickhill111 — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/callback/DocumentType.java (hunk 2 above)
Comment (paul-pearson) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/callback/DocumentType.java]: Done

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SendToFirstTierActions.java (hunk 3 above)
Comment (paul-pearson) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SendToFirstTierActions.java]: Done

**Review by nickhill111 — APPROVED** [review]

**Review by DGordonV1 — APPROVED** [review]

**Pull request merged**